src/components/SettingsFab.tsx

```diff
@@ -0,0 +1,35 @@
+import { IonFab, IonFabButton, IonIcon } from '@ionic/react';
+import { useDispatch, useSelector } from 'react-redux';
+import { menu } from 'ionicons/icons';
+import { Pages } from '../Models/Enums';
+import './TabBar.scss';
+import './SettingsFab.scss';
+import { selectRouter, setPage } from '../features/Routing/RouterStore';
+import React from 'react';
+const SettingsFab: React.FC = () => {
+    const dispatch = useDispatch();
+    const page = useSelector(selectRouter);
+
+    const handleClick = (button: Pages) => {
+        console.log(button);
```

Comment:
@brooke-k  do we need to keep this `console.log()` here?

Reply:
Removed it in the latest commit. Not sure why it was there, but it wasn't doing anything and I double-checked to make sure that nothing broke when I removed it - nothing did, so it's gone now.